client/ayon_houdini/plugins/publish/extract_active_view_thumbnail.py

```diff
@@ -17,7 +18,8 @@ class ExtractActiveViewThumbnail(plugin.HoudiniExtractorPlugin,
     """
     order = pyblish.api.ExtractorOrder + 0.49
     label = "Extract Active View Thumbnail"
-    families = ["workfile"]
+    families = ["workfile",
+                "review"]
```

Comment:
My feeling, this is more of a "workaround" than a "proper solution".

Additionally, a snapshot from the sceneview may not look the same as the first frame of the review.
This is not for review only but also render and image sequence products.

What about using the first frame as the thumbnail? (we'll need to convert it to `png` or `jpg`).

Reply:
Using the first frame as the thumbnail could also work. The input from our artists was that it would be great to be able to choose a specific frame. But you are right. In the end that will not look like the real render by using the screenshot of the viewport. So perhaps adding a option in the publisher to be able to select which frame should be used as thumbnail (not only the middle frame of the sequence) could be an option?

Reply:
feel free to create an issue for it on ayon-core.